lib/util/util_unlink.c

```diff
@@ -0,0 +1,31 @@
+/*
+   Copyright (C) 2017  Statoil ASA, Norway.
+
+   The file 'util_unlink.c' is part of ERT - Ensemble based Reservoir Tool.
+
+   ERT is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or
+   FITNESS FOR A PARTICULAR PURPOSE.
+
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+   for more details.
+*/
+
+#ifdef ERT_WINDOWS
+  #include <io.h>
+#else
+  #include <unistd.h>
+#endif
+
+int util_unlink(const char * filename) {
```

Comment:
Have tried to do these tests by looking for the existence of a specific function - i.e. `unlink( )` - instead of the catch all `ERT_WINDOWS` - see e.g.: https://github.com/Statoil/libecl/blob/master/lib/util/util.c#L3329